index.html

```diff
@@ -0,0 +1,92 @@
+<!doctype html>
+<html lang="de">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Quiz App</title>
+    <link rel="stylesheet" href="css/styles.css">
+</head>
+<body>
+<header class="header-main">
+    <h1 class="header-main--title">Quiz App</h1>
+</header>
+<main class="main-content">
+    <section class="main-container main-content--quiz">
+        <article class="card">
+            <header class="card-header">
+                <h2 class="card-header-title">Here the theme title</h2>
+                <span class="card-header-bookmark"><a href="#" title="bookmark this"></a></span>
+            </header>
+            <div class="card-content-question">
+                <p>
+                    Here comes the question with a litle description.<br />
+                    Lorem ipsum dolor sit amet, consectetur adipisicing elit. Aliquid amet commodi dicta dignissimos distinctio dolore eligendi esse illo iusto laboriosam magnam mollitia non!
+                </p>
+            </div>
+            <div class="card-content-userinput">
+                <form action="" method="post">
+                    <span class="card-answers"><input type="radio" name="answer" id="answer-one" value="answer one"><label for="answer-one" class="card-answer--text">This is the first answer</label></span>
+                    <span class="card-answers"><input type="radio" name="answer" id="answer-two" value="answer two"><label for="answer-two" class="card-answer--text">This is the second answer</label></span>
+                    <span class="card-answers"><input type="radio" name="answer" id="answer-three" value="answer three"><label for="answer-three" class="card-answer--text">This is the third answer</label></span>
+                    <button class="btn btn-primary" type="submit" aria-label="send">Check my answer</button>
```

Comment:
```suggestion
<button class="btn btn-primary" type="submit">Check my answer</button>
```
"Check my answer" is already descriptive, that´s fine to use for a11y.